File: HELLO_PYTHON/day06_3/mysql_insert.py
import pymysql
 
conn = pymysql.connect(host='127.0.0.1',port=3305, user='root', password='python',
                       db='python', charset='utf8')
 
 
#파이썬의 cursor는 자바의 statement와 비슷(자바의 커서와는 다르다)
# curs = conn.cursor(pymysql.cursors.DictCursor)  #딕셔너리(json형태)로 가져오는 법 
curs = conn.cursor()
#""" a """ 따음표 3개로 묶어놓으면 띄워쓰기해도 자동으로 인식해준다
sql = "INSERT INTO emp VALUES(%s, %s, %s, %s)"
curs.execute(sql,(4,'4','4','4'))
conn.commit()

curs.close()
conn.close()

File: HELLO_NODE2/mysql_insert.js
var mysql = require('mysql');

var connection = mysql.createConnection({
	host: 'localhost',
	user: 'root',
	port: 3305,
	password: 'python',
	database: 'python'
});

connection.connect(); // mysql과 연결

//var sql = "delete from emp where e_id='2'";
var sql = "update emp set addr = '5' where e_id='2'";

//var sql = "insert into emp values('2','1','1','1')";
connection.query(sql,(error,result)=>{
	console.log(result.affectedRows);
});

var sql2 = 'select * from emp';
connection.query(sql2, function(err, rows, fields) {
	if (err) {
		console.error('error connecting: ' + err.stack);
	}
	console.log(rows);
});
connection.end();

